platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functionnal/SOCNotificationIntranetMentionTestIT.java

```diff
@@ -0,0 +1,281 @@
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomString;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_ACTIVITY_MENTION_USER;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+@Tag("sniff")
+@Tag("social")
+
+public class SOCNotificationIntranetMentionTestIT extends Base {
+    NavigationToolbar navigationToolbar;
+    AddUsers addUsers;
+    IntranetNotification intranetNotification;
+    ManageLogInOut manageLogInOut;
+    MyNotificationsSetting myNotificationsSetting;
+    HomePagePlatform homePagePlatform;
+    ConnectionsManagement connectionsManagement;
+    ActivityStream activityStream;
+    UserAddManagement userAddManagement;
+    ArrayList<String> arrayUser;
+    NotificationActivity notificationActivity;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        navigationToolbar = new NavigationToolbar(this);
+        addUsers = new AddUsers(this);
+        intranetNotification = new IntranetNotification(this);
+        manageLogInOut = new ManageLogInOut(this);
+        myNotificationsSetting = new MyNotificationsSetting(this);
+        homePagePlatform = new HomePagePlatform(this);
+        connectionsManagement = new ConnectionsManagement(this);
+        userAddManagement = new UserAddManagement(this);
+        activityStream = new ActivityStream(this);
+        if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+            $(ELEMENT_SKIP_BUTTON).click();
+        }
+        manageLogInOut.signInCas(PLFData.DATA_USER1, "gtngtn");
+    }
+    /*Step Number: 1
+		 *Step Name:
+		 *Step Description:
+			- Login with User B
+			- Click the notification icons in the top navigation
+			- Check the notification list
+		 *Input Data:
+
+		 *Expected Outcome:
+			- The Mention notification is displayed in the list*/
+    /*Step number: 2
+		 *Step Name:
+		 *Step Description:
+			- Check the notification message
+		 *Input Data:
+
+		 *Expected Outcome:
+			The notification message is : $AVATAR$USER has mentioned you $ACTIVITY$DATEWhere :
+			- $AVATAR is the thumbnail of User A
+			- $USER is User A
+			- $ACTIVITY is the activity title/message
+			- $DATE is the date of the notification*/
+    /*Step Number: 1
+		 *Step Name:
+		 *Step Description:
+			- Login with User B
+			- Click the notification icons in the top navigation
+			- Check the notification list
+		 *Input Data:
+
+		 *Expected Outcome:
+			- The Mention notification is displayed in the list*/
+
+    /*Step number: 2
+     *Step Name:
+     *Step Description:
+        - Click the notification
+     *Input Data:
+
+     *Expected Outcome:
+        - The user is redirected to the activity viewer with all comment expanded.*/
+    @Test
+    public void test01_CheckMentionNotificationInActivityMessage() {
```

Comment:
This function contains 2 tests, you must split them

Reply:
done